tensorboard/webapp/app_routing/types.ts

```diff
@@ -38,6 +38,21 @@ export enum RouteKind {
   NOT_SET,
 }
 
+export type DashboardRoute =
+  | RouteKind.EXPERIMENT
+  | RouteKind.COMPARE_EXPERIMENT
+  | RouteKind.CARD;
+
+export function isDashboardRoute(
```

Comment:
Small thought that this might actually be nice as its own selector....just a thought.

Reply:
I thought about that too but I thought it read kinda weirdly. WDYT?
Imagine
```
private maybeFetchHparamsMetadata(
    experimentId: string
  ): Observable<HparamsAndMetadata> {
    return this.store.select(getEnableHparamsInTimeSeries).pipe(
      withLatestFrom(this.store.select(getIsDashboardRoute)),
      switchMap(([hparamsInTimeSeries, isDashboardRoute]) => {
        if (hparamsInTimeSeries && isDashboardRoute) {
          return of({
            hparamSpecs: [],
            metricSpecs: [],
            runToHparamsAndMetrics: {},
          });
        }
        ...

```

Reply:
I changed my mind. I think it is better this way.

Reply:
😂 alright then